site/content/en/docs/tasks/run/multikueue/plain_pods.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Run Plain Pod in Multi-Cluster"
+linkTitle: "Plain Pod"
+weight: 2
+date: 2025-02-17
+description: >
+  Run a MultiKueue scheduled Plain Pod.
+---
+
+## Before you begin
+
+1. Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+2. For the ease of setup and use we recommend using at least Kueue v0.11.0.
+
+3. Follow steps in [Run Plain Pods](/docs/tasks/run/plain_pods/#before-you-begin)
+to learn how to enable and configure the `v1/pod` integration.
```

Comment:
```suggestion
to learn how to enable and configure the `pod` integration.
```

Isn't this actually `Pod`?
https://github.com/kubernetes-sigs/kueue/blob/8b4affc551012a8b500d2f16a12acd1e282698b5/config/components/manager/controller_manager_config.yaml#L57

Reply:
I also updated the single cluster Plain Pod steps as it had the same incorrect naming for the `pod` integration